src/home/home.ts

```diff
@@ -416,6 +443,7 @@ export function showSelectedMessage(): void {
     readingPane.className = readingPane.className.replace(' new-mail', '');
     (document.getElementsByClassName('tb-item-new-mail')[0] as HTMLElement).style.display = 'inline-flex';
     (document.getElementsByClassName('tb-item-mark-read')[0] as HTMLElement).style.display = 'none';
+    document.getElementById('toolbar_align').style.display = '';
```

Comment:
Why display is set to empty?

Reply:
It behaves according to elements nature.